src/components/card/nz-card.component.ts

```diff
@@ -51,11 +52,31 @@ import {
   ]
 })
 export class NzCardComponent {
+  private _loading = false;
+  private _noHovering = false;
+
   @Input() @HostBinding('class.ant-card-bordered') nzBordered = true;
-  @Input() nzLoading = false;
-  @Input() @HostBinding('class.ant-card-no-hovering') nzNoHovering = false;
   @ContentChild('title') title: TemplateRef<any>;
   @ContentChild('extra') extra: TemplateRef<any>;
   @ContentChild('body') body: TemplateRef<any>;
   @HostBinding('class.ant-card') _nzCard = true;
+
+  @Input()
+  set nzLoading(value: boolean) {
+    this._loading = toBoolean(value);
+  }
+
+  get nzLoading(): boolean {
+    return this._loading;
+  }
+
+  @Input()
+  @HostBinding('class.ant-card-no-hovering')
```

Comment:
Theoretically `@HostBinding()` should on getter, but TypeScript requires all decorators on same part.